build.gradle.kts

```diff
@@ -17,6 +26,30 @@ dependencies {
     testImplementation("io.mockk:mockk:1.13.8")
 }
 
+tasks.withType<Jar> {
+    manifest {
+        attributes(
+            "Main-Class" to "app.MainKt"
+        )
+    }
+}
+
+tasks.register<Jar>("fatJar") {
+    archiveBaseName.set("task-manager")
+    from(sourceSets.main.get().output)
+    dependsOn(configurations.runtimeClasspath)
+
+    manifest {
+        attributes["Main-Class"] = "app.MainKt"
+    }
+}
+
```

Comment:
In the fatJar task above, may I know what the dependsOn(configurations.runtimeClasspath)? Because dependsOn(configurations.runtimeClasspath) doesn't actually include the dependencies in the JAR; it just ensures they are available at build time. Which might not be the intended purpose.